### DISCUSSION TITLE
Can I use the vercel/og package outside vercel infra? Is this part of Next.js or is it vercel infra related?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#41367] [Help]
Can I use the vercel/og package outside vercel infra? Is this part of Next.js, or is it vercel infra related?

I have many projects in vercel and I will use it a lot, but I also have some clients in AWS or Azure, and of course, I would like to use it in their infra as well.

Thanks guys!!

**Comment by gaston-flores** [comment]
I don't think it's possible to use `@vercel/og` outside of vercel, as it uses its Edge runtime. However, `og` uses [satori](https://github.com/vercel/satori) under the hood, which runs just fine in Node.
